src/cpu-info.spec.ts

```diff
@@ -1,20 +1,18 @@
-import { getCpuInfo } from './cpu-info';
+import { CpuInfo } from './cpu-info';
 
-describe('getCpuInfo', () => {
+describe('CpuInfo', () => {
   it('should get correct CPU information when available', () => {
```

Comment:
I have a suggestion about this "correct" test naming.

The word "correct" in the test case name somewhat vague and does not provide specific information about what the test is verifying. A good practice in naming test cases is to be as descriptive as possible about what the test is doing. We don't have incorrect work for `getCpuInfo`.

So my suggestion here will be to have 2 tests, and one will have something like:
"should get CPU information when available."
"should get 'undefined' CPU information when not available."